model/Database.php

```diff
@@ -992,5 +992,42 @@ function checkAdmin($user_id)
         return $statement->fetch(PDO::FETCH_ASSOC);
     }
 
+    /**
+     * update the database to store the path of attachments
+     * @param String $filePath name of the file/attachments
+     * @param int $itemId id of the item to where file paths are going to be stored
+     * @author Laxmi
+     */
+    function updateAttachments($filePath, $itemId)
```

Comment:
Could this function name be specific to the item table? Oleg also made a similar function but it updates the itemTutorYear table

Reply:
sure I will make a quick change. Thank you